bokehjs/src/lib/models/comparisons/nan_sorter.ts

```diff
@@ -0,0 +1,36 @@
+import {Comparison} from "./comparison"
+import type * as p from "core/properties"
+
+export namespace NanSorter {
+  export type Attrs = p.AttrsOf<Props>
+
+  export type Props = Comparison.Props & {
+    ascending_first: p.Property<boolean>
+  }
+}
+
+export interface NanSorter extends NanSorter.Attrs {}
+
+export class NanSorter extends Comparison {
```

Comment:
I would rename this to `NaNSorter`, to match the naming convention with `NaN`.

Reply:
There's also often `nan` or `NAN` and IME in a case like this folks usually let the CamelCasing take precedence. It's also slightly easier to type. I'll leave it as is unless there is a strenuous request for more discussion.